src/ir/context.rs

```diff
@@ -348,14 +348,31 @@ impl<'ctx> BindgenContext<'ctx> {
         let is_template_instantiation =
             is_type && item.expect_type().is_template_instantiation();
 
-        // Be sure to track all the generated children under namespace, even
-        // those generated after resolving typerefs, etc.
+        // Ensure that every item (other than the root module) is in a module's
+        // children list.
```

Comment:
Maybe point to the issue this is fixing or something? it may be slightly counter-intuitive otherwise.